### ISSUE TITLE
Standardize dollar format

### ISSUE THREAD

**Issue by annemeeker** [codeforboston/windfall-elimination#119]
$1,234.00

**Comment by thadk** [comment]
I recommend:
https://www.npmjs.com/package/d3-format
```
const value = 1000
const pretty = d3.format(",.2f")(value)
console.log(pretty)
> $1,000.00
```

**Comment by markswinimer** [comment]
Working on this.

**Comment by thadk** [comment]
This was mostly well addressed by #175 but @markswinimer said he might want to continue adding $ and "," to the manual entry table.
